.changes/unreleased/Features-20231107-135635.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
+body: Remove legacy logger
```

Comment:
I think we don't use this on cloud anymore, do we need to double check?

Reply:
@MichelleArk checked and they have all moved to event logging